syntax/crystal.vim

```diff
@@ -16,7 +16,7 @@ endif
 syn iskeyword @,48-57,_,192-255,@-@
 
 " eCrystal Config
-if exists('g:main_syntax') && g:main_syntax ==# 'ecrystal'
+if exists('g:main_syntax') && g:main_syntax == 'ecrystal'
```

Comment:
Please avoid `==` for comparing strings because its behavior is depending on `ignorecase` option. Using `==#` or `==?` is better to avoid confusion.